lib/tasks/multitenant_db_merge.rake

```diff
@@ -0,0 +1,344 @@
+desc 'Merge an ext daria db into main db'
+task multitenant_db_merge: :environment do
+  raise 'SET MIGRATION_FUND_ID ENV VAR' if not ENV['MIGRATION_FUND_ID']
+  raise 'SET MIGRATION_DB_URL ENV VAR' if not ENV['MIGRATION_DB_URL']
+
+  # Make absolutely sure the new fund has been created!
+  fund = Fund.find ENV['MIGRATION_FUND_ID']
+  @fund_id = fund.id
+
+  # Scope funds and turn off papertrail
+  ActsAsTenant.current_tenant = fund
+  PaperTrail.enabled = false # We're porting these over more directly and don't need extra versions
+
+  # Database we're migrating things into
+  def connect_to_target_db
+    ActiveRecord::Base.establish_connection(Rails.env.to_sym)
+  end
+
+  # Database we're migrating things out of
+  def connect_to_migration_db
+    ActiveRecord::Base.establish_connection(
+      adapter: 'postgresql',
+      encoding: 'unicode',
+      pool: 10,
+      url: ENV['MIGRATION_DB_URL']
+    )
+    ActiveRecord::Base.connection.execute(
+      "set session characteristics as transaction read only;"
+    )
+  end
+
+  puts "Migrating into fund_id #{@fund_id}..."
+
+  if Patient.count() > 0
+    raise 'Possibly already ported? patients belonging to that fund_id are found'
```

Comment:
It's frankly not pretty! But we'd probably set tenant and `Patient.destroy_all`, `Clinic.destroy_all`, etc to clean up. To avoid this we can dry run it in a local environment.  I agree that it's less than ideal but since this code will outlive its usefulness quickly, felt fine to me.

That said, we COULD stick this in a `begin/rescue` block and do the cleanout if we wanted to, though I think if this DID choke I'd want to open up a rails console and clean it up by hand, rather than letting therobots do it. idk, what do you think?

Reply:
I want to defer to @xmunoz's comfort here, since the plan is to have y'all running it!

Reply:
In the spirit of expedience, let's assume that this is okay, and if we have concerns after the first few we can retcon that into the script?